pkg/services/cloudprovider/csi.go

```diff
@@ -306,6 +302,14 @@ func VSphereCSINodeContainer(image string) corev1.Container {
 				Name:  "VSPHERE_CSI_CONFIG",
 				Value: "/etc/cloud/csi-vsphere.conf",
 			},
+			{
+				Name:  "LOGGER_LEVEL",
+				Value: "PRODUCTION",
+			},
+			{
+				Name:  "X_CSI_DEBUG",
```

Comment:
Do we want debug on by default?

Reply:
IIRC `X_CSI_DEBUG` enables logging entries of GoCSI. perhaps @codenrhoden can confirm

Reply:
yes and no. setting `X_CSI_DEBUG=true` would set the log level to `DEBUG`, which will enable a *lot* of stuff, and also enables request/response logging. 

From the [GoCSI](https://github.com/rexray/gocsi) README:
```
A true value is equivalent to:
X_CSI_LOG_LEVEL=debug
X_CSI_REQ_LOGGING=true
X_CSI_REP_LOGGING=true
```
So it depends on if you want all those things turned on.
Request and response logging can be independentently controlled, and the default `X_CSI_LOG_LEVEL` is `WARN`. This could be changed to `INFO` if you want it more verbose by default.

Reply:
OK, let's leave DEBUG=false
and set X_CSI_LOG_LEVEL=info instead.

Reply:
If you are doing that, which I think makes sense, I would also recommend setting it for the controller as well as the node. Looks like it is only set for the node right now.

Reply:
done